poseidonslow/poseidon.go

```diff
@@ -0,0 +1,89 @@
+package poseidonslow
```

Comment:
Was not sure, should I delete this code or not. So decided to keep it, as it has done in https://github.com/iden3/circomlib